backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -400,4 +487,12 @@ public void onEvent(BacktraceResult backtraceResult) {
             }
         };
     }
+
+    private void addReportAttachments(BacktraceReport report) {
```

Comment:
Am I correct that we will add this attachment to every report? Is this the expected behavior?

Reply:
Yes and yes